towncrier/newsfragments/1214.enhancement

```diff
@@ -0,0 +1 @@
+Move rspamd, fetchmail and unbound services into core and optional
```

Comment:
Same [comment](https://github.com/Mailu/Mailu/pull/1221/files#r334758529) on file extension as for PR #1221.

Reply:
Renamed to `.misc`